cuppa/src/main/java/org/forgerock/cuppa/Runner.java

```diff
@@ -50,38 +51,75 @@
             .setTestClass(Cuppa.class)
             .setDescription("")
             .build();
+    private static final List<Function<TestBlock, TestBlock>> DEFAULT_CORE_TEST_TRANSFORMS =
+            Arrays.asList(new OnlyTestBlockFilter(), new EmptyTestBlockFilter());
 
     private final List<Function<TestBlock, TestBlock>> coreTestTransforms;
     private final Configuration configuration;
+    private final Options runState;
```

Comment:
I think I would prefer to call this `runOptions` or `runParameters`, but I'm fine with the current name if you like it.